### ISSUE TITLE
Open the app from "base" links in mobile

### ISSUE THREAD

**Issue by Tsurha** [IsraelHikingMap/Site#1561]
## Bug/Feature (I'm not sure which...)
Launch the app when from a mobile phone clicking a relevant link like: https://israelhiking.osm.org.il/

## What I expect to happen
To have the app open at the link's location with me logged in and all my defaults loaded

## What really happened
I got the phone browser opened like this:
![image](https://user-images.githubusercontent.com/49042192/127969920-037b6c0d-2d0f-4442-830f-e1866af51913.png)

## What I did that caused the issue - step by step
Clicked the link and was surprised by the behavior 

## My environment
![image](https://user-images.githubusercontent.com/49042192/127970322-024b98c3-6fbc-4fd5-b702-c0164bcd2d7b.png)

## Things I think the developers should know, images, links etc.
I imagine its having the app register the IHM domain in the OS as related to the app. But what do I know...

**Comment by HarelM** [comment]
TL;DR: it's a not so interesting bug.
This is basically working for links which have "data" in them such as cloud save, map location etc.
The above link is similar to opening the app by using the apps menu in your phone, so in theory it doesn't add any thing interesting to the data in the app.
Having said that, I understand the requirement from the user experience point of view and I'll see how complicated it is to support it.
